test/testdrive/kafka-avro-upsert-sinks.td

```diff
@@ -279,6 +279,27 @@ $ kafka-create-topic topic=non-keyed-input
   INTO KAFKA CONNECTION kafka_conn (TOPIC 'not-enforced-sink-${testdrive.seed}') KEY (a) NOT ENFORCED
   FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn ENVELOPE UPSERT
 
+# Send a create, an update, and a delete for two separate keys.
+$ kafka-ingest format=avro topic=non-keyed-input schema=${schema}
+{"array":[{"data":{"a":1,"b":1},"time":1,"diff":1}]}
+{"array":[{"data":{"a":2,"b":1},"time":1,"diff":1}]}
+{"array":[{"data":{"a":1,"b":1},"time":2,"diff":-1}]}
+{"array":[{"data":{"a":2,"b":1},"time":2,"diff":-1}]}
+{"array":[{"data":{"a":1,"b":2},"time":2,"diff":1}]}
+{"array":[{"data":{"a":2,"b":2},"time":2,"diff":1}]}
+{"array":[{"data":{"a":1,"b":2},"time":3,"diff":-1}]}
+{"array":[{"data":{"a":2,"b":2},"time":3,"diff":-1}]}
+{"com.materialize.cdc.progress":{"lower":[0],"upper":[10],"counts":[{"time":1,"count":1}, {"time":2,"count":2}, {"time":3,"count":1}]}}
```

Comment:
How come the sum of the `counts` does not match the number of messages ingested not the difference between upper and lower?

Reply:
I think the counts just need to be doubled. They are meant to be the number of non-zero updates for each time, and it looks like maybe they were for some single-key version of the test, and when adding a second key in they were not doubled.